wayland-display-core/src/comp/input.rs

```diff
@@ -60,7 +61,7 @@ impl State {
                                 && !modifiers.alt
                                 && !modifiers.logo
                             {
-                                match handle.modified_sym() {
+                                match u32::from(handle.modified_sym()) {
                                     keysyms::KEY_Tab => {
```

Comment:
So this would become 
```suggestion
                                    Keysym::Tab => {
```

and so on...